keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/PassThruController.kt

```diff
@@ -0,0 +1,45 @@
+package com.netflix.spinnaker.keel.rest
+
+import com.netflix.spinnaker.keel.clouddriver.CloudDriverService
+import com.netflix.spinnaker.keel.clouddriver.model.ActiveServerGroup
+import com.netflix.spinnaker.keel.clouddriver.model.toActive
+import com.netflix.spinnaker.keel.core.api.DEFAULT_SERVICE_ACCOUNT
+import kotlinx.coroutines.runBlocking
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+/**
+ * This controller passes through calls to downstream services (e.g., clouddriver).
+ *
+ * It's used only for testing.
+ */
+@RestController
+@ConditionalOnProperty("tests.passthru", matchIfMissing = false)
+@RequestMapping(path=["/test"])
+class PassThruController(
```

Comment:
I wrote this controller to help me test this new logic, and figured it might be useful to have it in the codebase, but I can zap it if others feel differently.

Reply:
I feel like we can zap it when we start using the new call.